src/test/java/mpack/Assignement4.java

```diff
@@ -0,0 +1,95 @@
+//Dynamic Xpath
+package mpack;
+
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.safari.SafariDriver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class Assignement4 {
+	static WebDriver driver =null;
+
+	public static WebDriver startbrowser(String browser_name)
+	{
+
+		if(browser_name.equalsIgnoreCase("chrome"))
+		{
+			WebDriverManager.chromedriver().setup();
+		    driver = new ChromeDriver();
+		}
```

Comment:
We can move this in separate  method and pass browser name in argument .
hence , it will call required browser 
In case you want to do cross browser testing 
add some loop and if else 
pass browser name as argument and single code will run on all browser

Reply:
Created separate method in the base class and accessing it wherever required in the argument